src/content/app/species-selector/components/popular-species-list/PopularSpeciesList.tsx

```diff
@@ -43,8 +44,10 @@ const PopularSpeciesList = () => {
 
   const onPopularSpeciesButtonClick = (species: PopularSpecies) => {
     dispatch(setPopularSpecies(species));
-    dispatch(setModalView('popular-species-genomes'));
```

Comment:
Now that we are getting rid of the modal view mechanism, we should remove the code that relied on it (from redux slice, and from the components that used the selectors that were reading that view).